doc/api.md

```diff
@@ -125,20 +166,144 @@ Favorites aren't displayed to other users, so "user_id" and "screen_name". So se
 #### Unsupported parameters
 * user_id
 * screen_name
-* cursor 
+* cursor
 
 Friendica doesn't allow showing followers of other users.
 
+---
+### friendica/activity/<verb>
+#### parameters
+* id: item id
+
+Add or remove an activity from an item.
+'verb' can be one of:
+- like
+- dislike
+- attendyes
+- attendno
+- attendmaybe
+
+To remove an activity, prepend the verb with "un", eg. "unlike" or "undislike"
+Attend verbs disable eachother: that means that if "attendyes" was added to an item,

```

Comment:
could you remove the linebreak? We try to have "One sentence per line" in the docs ;-)